packages/connect-explorer/src/js/data/methods/management/recoverDevice.ts

```diff
@@ -0,0 +1,66 @@
+const name = 'recoveryDevice';
+const docs = 'methods/recoverDevice.md';
```

Comment:
It looks like this variable `docs` is never used.

Reply:
fixed